WooCommerce/Classes/Authentication/Navigation Exceptions/JetpackSetupWebViewModel.swift

```diff
@@ -13,127 +12,54 @@ final class JetpackSetupWebViewController: UIViewController {
     /// The closure to trigger when Jetpack setup completes.
     private let completionHandler: (String?) -> Void
 
-    /// Main web view
-    private lazy var webView: WKWebView = {
-        let webView = WKWebView(frame: .zero)
-        webView.translatesAutoresizingMaskIntoConstraints = false
-        webView.navigationDelegate = self
-        return webView
-    }()
-
-    /// Progress bar for the web view
-    private lazy var progressBar: UIProgressView = {
-        let bar = UIProgressView(progressViewStyle: .bar)
-        bar.translatesAutoresizingMaskIntoConstraints = false
-        return bar
-    }()
-
-    /// Strong reference for the subscription to update progress bar
-    private var progressSubscription: AnyCancellable?
-
     /// The email address that the user uses to authorize Jetpack
     private var authorizedEmailAddress: String?
 
     init(siteURL: String, analytics: Analytics = ServiceLocator.analytics, onCompletion: @escaping (String?) -> Void) {
         self.siteURL = siteURL
         self.analytics = analytics
         self.completionHandler = onCompletion
-        super.init(nibName: nil, bundle: nil)
-    }
-
-    required init?(coder: NSCoder) {
-        fatalError("init(coder:) has not been implemented")
-    }
-
-    override func viewDidLoad() {
-        super.viewDidLoad()
-        configureNavigationBar()
-        configureWebView()
-        configureProgressBar()
-        startLoading()
-    }
-
-    override func viewWillDisappear(_ animated: Bool) {
-        super.viewWillDisappear(animated)
-        if isMovingFromParent {
-            analytics.track(event: .LoginJetpackSetup.setupDismissed(source: .web))
-        }
-    }
-}
-
-private extension JetpackSetupWebViewController {
-    func configureNavigationBar() {
-        title = Localization.title
-    }
-
-    func configureWebView() {
-        view.addSubview(webView)
-        NSLayoutConstraint.activate([
-            view.leadingAnchor.constraint(equalTo: webView.leadingAnchor),
-            view.trailingAnchor.constraint(equalTo: webView.trailingAnchor),
-            view.safeTopAnchor.constraint(equalTo: webView.topAnchor),
-            view.bottomAnchor.constraint(equalTo: webView.bottomAnchor),
-        ])
     }
 
-    func configureProgressBar() {
-        view.addSubview(progressBar)
-        NSLayoutConstraint.activate([
-            view.leadingAnchor.constraint(equalTo: progressBar.leadingAnchor),
-            view.trailingAnchor.constraint(equalTo: progressBar.trailingAnchor),
-            view.safeTopAnchor.constraint(equalTo: progressBar.topAnchor)
-        ])
-    }
+    // MARK: - `PluginSetupWebViewModel` conformance
+    var title: String { Localization.title }
```

Comment:
super nit: this can also be a `let`
```suggestion
    let title = Localization.title
```

Reply:
Fixed in 1899b57cf9a728acce705d2eb605f2e175a09079 👍